app/api/__init__.py

```diff
@@ -10,6 +11,13 @@
 # users
 api.route(UserList, 'user_list', '/users')
 api.route(UserDetail, 'user_detail', '/users/<int:id>')
```

Comment:
@enigmaeth Are you sure this works, did you try to retrieve a user via those endpoints. I'm not sure it will work as there is no code similar to 
```
def before_get_object(self, view_kwargs):
        if view_kwargs.get('computer_id') is not None:
            try:
                computer = self.session.query(Computer).filter_by(id=view_kwargs['computer_id']).one()
            except NoResultFound:
                raise ObjectNotFound({'parameter': 'computer_id'},
                                     "Computer: {} not found".format(view_kwargs['computer_id']))
            else:
                if computer.person is not None:
                    view_kwargs['id'] = computer.person.id
                else:
                    view_kwargs['id'] = None
``` on http://flask-rest-jsonapi.readthedocs.io/en/latest/quickstart.html#first-example
Please correct me If I'm wrong.